{
	"documenttitle": {
		"text": "Red Hat Enhancement Advisory: Red Hat Advanced Cluster Management for Kubernetes version 2.2 images",
		"lang": "en"
	},
	"DocumentType": "Security Advisory",
	"documentpublisher": {
		"type": "Vendor",
		"ContactDetails": "secalert@redhat.com",
		"IssuingAuthority": "Red Hat Product Security"
	},
	"documenttracking": {
		"identification": {
			"ID": "RHEA-2021:0729"
		},
		"Status": "Final",
		"Version": "1",
		"revisionhistory": {
			"revision": {
				"Number": "1",
				"Date": "2021-10-28T05:02:00Z",
				"Description": "Current version"
			}
		},
		"InitialReleaseDate": "2021-03-04T15:08:00Z",
		"CurrentReleaseDate": "2021-10-28T05:02:00Z",
		"generator": {
			"Engine": "Red Hat SDEngine 3.22.0",
			"Date": "2023-09-15T01:25:00Z"
		}
	},
	"documentnotes": {
		"note": [
			{
				"text": "\nRed Hat Advanced Cluster Management for Kubernetes 2.2.0 General Availability\nrelease\n    ",
				"title": "Topic",
				"type": "Summary",
				"ordinal": "1",
				"lang": "en"
			},
			{
				"text": "\nRed Hat Advanced Cluster Management for Kubernetes 2.2.0 images\n\nRed Hat Advanced Cluster Management for Kubernetes provides the\ncapabilities to address common challenges that administrators and site\nreliability engineers face as they work across a range of public and\nprivate cloud environments. Clusters and applications are all visible and\nmanaged from a single console—with security policy built in.\n\nThis advisory contains the container images for Red Hat Advanced Cluster\nManagement for Kubernetes. See the following Release Notes documentation,\nwhich will be updated shortly for this release, for details about the enhancements in this release:\n\nhttps://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_mana\ngement_for_kubernetes/2.2/html/release_notes/\n\nBug fixes: \n* multicluster-operators-standalone-subscription pod OOMkilled due to memory usage (BZ #1867467)\n* RFE x509: certificate signed by unknown authority when using Git Channel with self-signed certs (BZ #1891510)\n* RHACM 2.2.0 images (BZ #1893876)\n* Configuring RBAC for limiting visibility (BZ #1896757)\n* IPv6 square brackets is marked invalid format in bare metal asset screen (BZ #1899147)\n* endpoint-observability-operator RBAC issues (BZ #1913447)\n* [OCP 4.6 | ACM 2.2 | BM IPI] failed to deploy MCH via operator - Cert manager container creation error (BZ# 1915503)\n* RHACM Observability installation failing for OCP4.5 (BZ #192496)\n    ",
				"title": "Details",
				"type": "General",
				"ordinal": "2",
				"lang": "en"
			},
			{
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use",
				"type": "Legal Disclaimer",
				"ordinal": "3",
				"lang": "en"
			}
		]
	},
	"documentdistribution": {
		"text": "Copyright © 2023 Red Hat, Inc. All rights reserved.",
		"lang": "en"
	},
	"aggregateseverity": {
		"text": "Moderate",
		"namespace": "https://access.redhat.com/security/updates/classification/"
	},
	"documentreferences": {
		"reference": [
			{
				"type": "Self",
				"URL": "https://access.redhat.com/errata/RHEA-2021:0729",
				"Description": "https://access.redhat.com/errata/RHEA-2021:0729"
			}
		]
	},
	"vulnerability": [
		{
			"ordinal": "1",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found in golang-github-buger-jsonparser. The Library API in buger jsonparser through 2019-12-04 allows attackers to cause a denial of service (infinite loop) via a delete call. The highest threat from this vulnerability is to system availability.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2020-03-09T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2020-10675",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.2/html-single/install/index#installing\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHEA-2021:0729"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2020-10675",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2020-10675"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2020-10675",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2020-10675"
					},
					{
						"URL": "https://github.com/buger/jsonparser/issues/188",
						"Description": "https://github.com/buger/jsonparser/issues/188"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2020-10675",
						"Description": "CVE-2020-10675"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1817733",
						"Description": "bz#1817733: CVE-2020-10675 golang-github-buger-jsonparser: infinite loop via a Delete call"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2020-03-19T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "2",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "node-fetch before versions 2.6.1 and 3.0.0-beta.9 did not honor the size option after following a redirect, which means that when a content size was over the limit, a FetchError would never get thrown and the process would end without failure. For most people, this fix will have a little or no impact. However, if you are relying on node-fetch to gate files above a size, the impact could be significant, for example: If you don't double-check the size of the data after fetch() has completed, your JS thread could get tied up doing work on a large file (DoS) and/or cost you money in computing.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2020-09-10T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2020-15168",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.2/html-single/install/index#installing\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHEA-2021:0729"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2020-15168",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2020-15168"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2020-15168",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2020-15168"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2020-15168",
						"Description": "CVE-2020-15168"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1882329",
						"Description": "bz#1882329: size of data after fetch() JS thread leads to DoS"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Low"
				}
			},
			"DiscoveryDate": "2020-09-18T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "3",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "In Helm before versions 2.16.11 and 3.3.2 there is a bug in which the `alias` field on a `Chart.yaml` is not properly sanitized. This could lead to the injection of unwanted information into a chart. This issue has been patched in Helm 3.3.2 and 2.16.11. A possible workaround is to manually review the `dependencies` field of any untrusted chart, verifying that the `alias` field is either not used, or (if used) does not contain newlines or path characters.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2020-09-18T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2020-15184",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.2/html-single/install/index#installing\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHEA-2021:0729"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2020-15184",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2020-15184"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2020-15184",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2020-15184"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2020-15184",
						"Description": "CVE-2020-15184"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1882357",
						"Description": "bz#1882357: CVE-2020-15184 helm: Chart.yaml is not properly sanitized lead to injection of unwanted information into chart"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Low"
				}
			},
			"DiscoveryDate": "2020-09-18T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "4",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "In Helm before versions 2.16.11 and 3.3.2, a Helm repository can contain duplicates of the same chart, with the last one always used. If a repository is compromised, this lowers the level of access that an attacker needs to inject a bad chart into a repository. To perform this attack, an attacker must have write access to the index file (which can occur during a MITM attack on a non-SSL connection). This issue has been patched in Helm 3.3.2 and 2.16.11. A possible workaround is to manually review the index file in the Helm repository cache before installing software.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2020-09-18T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2020-15185",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.2/html-single/install/index#installing\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHEA-2021:0729"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2020-15185",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2020-15185"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2020-15185",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2020-15185"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2020-15185",
						"Description": "CVE-2020-15185"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1882306",
						"Description": "bz#1882306: CVE-2020-15185 helm: write access to the index file allows an attacker to inject  bad chart into  repository"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Low"
				}
			},
			"DiscoveryDate": "2020-09-18T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "5",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "In Helm before versions 2.16.11 and 3.3.2 plugin names are not sanitized properly. As a result, a malicious plugin author could use characters in a plugin name that would result in unexpected behavior, such as duplicating the name of another plugin or spoofing the output to `helm --help`. This issue has been patched in Helm 3.3.2. A possible workaround is to not install untrusted Helm plugins. Examine the `name` field in the `plugin.yaml` file for a plugin, looking for characters outside of the [a-zA-Z0-9._-] range.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2020-09-18T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2020-15186",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.2/html-single/install/index#installing\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHEA-2021:0729"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2020-15186",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2020-15186"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2020-15186",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2020-15186"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2020-15186",
						"Description": "CVE-2020-15186"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1882302",
						"Description": "bz#1882302: CVE-2020-15186 helm: plugin names are not sanitized properly"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Low"
				}
			},
			"DiscoveryDate": "2020-09-18T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "6",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "In Helm before versions 2.16.11 and 3.3.2, a Helm plugin can contain duplicates of the same entry, with the last one always used. If a plugin is compromised, this lowers the level of access that an attacker needs to modify a plugin's install hooks, causing a local execution attack. To perform this attack, an attacker must have write access to the git repository or plugin archive (.tgz) while being downloaded (which can occur during a MITM attack on a non-SSL connection). This issue has been patched in Helm 2.16.11 and Helm 3.3.2. As a possible workaround make sure to install plugins using a secure connection protocol like SSL.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2020-09-18T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2020-15187",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.2/html-single/install/index#installing\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHEA-2021:0729"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2020-15187",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2020-15187"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2020-15187",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2020-15187"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2020-15187",
						"Description": "CVE-2020-15187"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1882300",
						"Description": "bz#1882300: CVE-2020-15187 helm: write access to the git repository or plugin archive causing causing a local execution attack"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2020-09-18T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "7",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "jsonparser 1.0.0 allows attackers to cause a denial of service (panic: runtime error: slice bounds out of range) via a GET call.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2020-12-06T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2020-35381",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.2/html-single/install/index#installing\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHEA-2021:0729"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2020-35381",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2020-35381"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2020-35381",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2020-35381"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2020-35381",
						"Description": "CVE-2020-35381"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1908451",
						"Description": "bz#1908451: CVE-2020-35381 jsonparser: GET call can lead to a slice bounds out of range"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Low"
				}
			},
			"DiscoveryDate": "2020-12-15T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "8",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "RedisGraph 2.x through 2.2.11 has a NULL Pointer Dereference that leads to a server crash because it mishandles an unquoted string, such as an alias that has not yet been introduced.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2020-12-21T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2020-35668",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.2/html-single/install/index#installing\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHEA-2021:0729"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2020-35668",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2020-35668"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2020-35668",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2020-35668"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2020-35668",
						"Description": "CVE-2020-35668"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1911331",
						"Description": "bz#1911331: CVE-2020-35668 redisgraph: NULL pointer dereference because it mishandles an unquoted string"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2020-12-23T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		}
	],
	"producttree": {}
}
